### DISCUSSION TITLE
Code for two start times per day

### DISCUSSION THREAD

**Discussion by ZontiOtto** [gopro/labs#780] [Q&A] [answered]
Maybe someone can help me make a GoPro Labs code for the following requirement.
A GoPro 8 should switch on at 06:00 AM and switch off at 09:00 AM, generate a time-lapse video during this time.
The whole thing should repeat from 04:00 PM to 07:00 PM.
Thanks for your help
Gerhard

**Comment by dnewman-gpsw** [comment]
This is pretty simple

```dT!06:00S!09:00E!16:00S!19:00E!R```

explained:
dT - default Timelapse
!06:00S - Sleep and Start at 6am.
!09:00E - End at 9am
!16:00S - Sleep and Start at 4pm.
!19:00E - End at 7pm
!R - repeat

You will require external power, as that is a lot of capture time.

**Reply by ZontiOtto** [reply, marked as answer]
Thank you very much for the quick reply and the help.
I am ashamed that I did not find the solution myself. I have tried many things but never used the "!R" command.
I think that's where I failed :-(
I use a power bank (Aukey PB-N37), which usually lasts me about 8-9 hours (non-stop) with a 5 second interval. That's almost always enough for my projects.
Thanks again and best regards from Wels in Austria Gerhard

**Reply by dnewman-gpsw** [reply]
This is some risk in that power bank, they can inappropriately shutoff (most phone chargers do.)

You should use an always on power bank like Voltaic V50.  Add a solar polar (as Voltaic brand) and it will run for a long time.)  https://voltaicsystems.com/v50/
